sys/net/sock/sock_util.c

```diff
@@ -28,15 +28,23 @@
 
 #include "net/sock/udp.h"
 #include "net/sock/util.h"
+#include "net/iana/portrange.h"
 
 #if defined(MODULE_DNS)
 #include "net/dns.h"
 #endif
 
+#ifdef MODULE_RANDOM
```

Comment:
Does this have to be guarded? Because the call to `random_uint32_range()` is not.

Reply:
The `sock_dtls` module will always pull in the `random` module, but sock_util may be used without `random`.